builder/base/twig/kss_builder_base_twig.js

```diff
@@ -73,9 +73,10 @@ class KssBuilderBaseTwig extends KssBuilderBase {
       };
       this.options.namespace.forEach(namespace => {
         // namespace should be of the form "namespace:path";
-        let tokens = namespace.split(':', 2);
-        if (tokens[1]) {
-          this.namespaces[tokens[0]] = path.resolve(tokens[1]);
+        let namespacePart = namespace.substr(0, namespace.indexOf(':'));
```

Comment:
```suggestion
        const namespacePart = namespace.substr(0, namespace.indexOf(':'));
```

Reply:
I just changed the logic, it was already `let` ;-)

Reply:
I saw it ^^ old code is not the cleanest.